test/filters.spec.js

```diff
@@ -3,6 +3,10 @@ var TestUtils = require("./utils");
 var check_juttle = TestUtils.check_sql_juttle;
 
 describe('test filters', function () {
+    before(function() {
+        TestUtils.init();
+        return TestUtils.loadTables();
```

Comment:
I could...but that would be so much slower: to write and delete the tables before every file.